industrial_robot_simulator/package.xml

```diff
@@ -25,4 +22,5 @@
   <run_depend>industrial_robot_client</run_depend>
```

Comment:
There is a launch dependency...[here](https://github.com/ros-industrial/industrial_core/blob/indigo-devel/industrial_robot_simulator/launch/robot_interface_simulator.launch)